src/test/java/calculator/Calculator.java

```diff
@@ -0,0 +1,47 @@
+package calculator;
+
+public class Calculator {
+    public boolean isBlank(String input) {
+        if(input.equals(" ") || input == null)
+            return true;
+        return false;
+    }
+    public String[] splitBlank(String str){
+        return str.split(" ");
+    }
+    public int calculate(int firstNum, int secondNum, char operator){
+        if(operator == '+')
+            return add(firstNum, secondNum);
+        if(operator == '-')
+            return subtract(firstNum, secondNum);
+        if(operator == '*')
+            return multiply(firstNum, secondNum);
+        if(operator == '/')
+            return divide(firstNum, secondNum);
+        throw new IllegalArgumentException();
+    }
+    public int add(int firstNum, int secondNum){
+        return firstNum + secondNum;
+    }
+    public int subtract(int firstNum, int secondNum) {
+        return firstNum - secondNum;
+    }
+    public int multiply(int firstNum, int secondNum) {
+        return firstNum * secondNum;
+    }
+    public int divide(int firstNum, int secondNum) {
+        return firstNum / secondNum;
+    }
+    public int getResult(String input){
+        if(isBlank(input))
+            throw new IllegalArgumentException();
+        return splitedNumCalculate(splitBlank(input));
+    }
+    public int splitedNumCalculate(String[] str) {
+        int result = Integer.parseInt(str[0]);
+        for(int i = 0; i < str.length - 2; i += 2) {
```

Comment:
반복 패턴을 잘 찾으셨습니다👍

한가지 Tip을 드리면 조건문을 함수로 만들면
코드가 명확해지고 읽기가 쉬워져요.
아래는 예시 참고하셔서 변경해 보시면 좋겠습니다😄
```suggestion
        for(isOperatiorPosition(index)) {
```

Reply:
제가 가이드를 잘못드렸습니다.😭
저 내용은 if문에 대한 조건을 함수로 만드는 내용인데 for문에는 적용하기 힘들어요.
참고만 해주시고 코드는 기존대로 하셔도 괜찮을거 같네요😅